actors/evm/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is code coming here in a follow-up? Otherwise what does this empty file do?

Reply:
Yes, (lots of) code coming here soon. It's currently a placeholder to make things build.